web_src/js/components/DiffCommitSelector.vue

```diff
@@ -95,6 +95,20 @@ export default {
       }
     });
 
+    // Add an eventlistener for shift keyup
+    // only is active if hoverActivated is true.
+    document.addEventListener('keyup', (evt) => {
```

Comment:
Why not reuse `this.$el.addEventListener('keyup', ...` ?

Reply:
If someone does keyup outside of the element i wanted to catch this, too.

Reply:
Hmm ... still don't get the point how could it happen.

The focus should always be in the "$el", if the focus is not in the "$el", I think the "$el" should be hidden, then no need to handle the keyup?

Reply:
Sounds reasonable - do you want to change it or should I do this (today evening German time)?

Reply:
Either is fine (I am going out soon, so if I have time half-day later I could also do the change)